packages/create-app/src/lib/tasks.ts

```diff
@@ -227,6 +227,9 @@ export async function moveAppTask(
           `Failed to move app from ${tempDir} to ${destination}: ${error.message}`,
         );
       })
+      .then(() => {
+        fs.writeFileSync(resolvePath(destination, '.nvmrc'), 'lts/iron');
```

Comment:
Would it not make more sense to just include the file, say here: https://github.com/backstage/backstage/tree/master/packages/create-app/templates/default-app, that way you don't have to create it?

Reply:
you can circumvent that, for example with `--force` https://git-scm.com/docs/git-add